.github/workflows/ci.yml

```diff
@@ -136,6 +136,19 @@ jobs:
           "setuptoolsShellHook &&
            pytest tests -v"
 
+  cc-wrapper-build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Pip3 backup
```

Comment:
No, this should make possible to use pip3 after installation of pip2, otherwise the installation of pip2 at least what I tried makes impossible to use  (uninstalls) pip3.
Right now the pip3 is not used there but I wanted to make sure that pip3 will be possible to use in case in the future we will need to use pip2 and pip3 at the same time.